nodejs/week1/avg.js

```diff
@@ -0,0 +1,26 @@
+const arguments = process.argv;
+
+let numbers = []; 
+
+for (let arg of arguments) { 
+    let num = Number(arg); 
+    if (!isNaN(num)) { 
+        numbers.push(num);
+    }
+}
+
+
+if (numbers.length === 0) {
+    console.log("add a number");
```

Comment:
This message should be clearer; tell the user that they need to provide valid numbers as parameters.